packages/builders/src/interactions/modals/Modal.ts

```diff
@@ -62,18 +69,28 @@ export class ModalBuilder implements JSONEncodable<APIModalInteractionResponseCa
 	 *
 	 * @param components - The components to add
 	 */
-	public addComponents(
+	public addActionRows(
```

Comment:
The JSDoc needs updating (can't request changes there). The JSDoc for `setActionRows` also needs updating.

> Adds action rows to this modal.